server/db/mysql/adapter.go

```diff
@@ -1518,16 +1528,19 @@ func (a *adapter) OwnTopics(uid t.Uid) ([]string, error) {
 	if err != nil {
 		return nil, err
 	}
+	defer rows.Close()
```

Comment:
In general, `defer` is less performant than a direct call. There is no need to convert all direct calls to `defer`. Please use `defer` only when it is really necessary and it indeed simplifies the code.